github/teams_discussion_comments.go

```diff
@@ -39,12 +39,12 @@ type DiscussionCommentListOptions struct {
 	Direction string `url:"direction,omitempty"`
```

Comment:
Please add `ListOptions` as an embedded struct to `DiscussionCommentListOptions`, as it appears to be missing.